other/csharp/convertRDF.cs

```diff
@@ -0,0 +1,56 @@
+using VDS.RDF.Ontology;
+using VDS.RDF.Parsing;
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace DigitalTwins_Samples
+{
+    public class DigitalTwinsConvertRDFSample
+    {
+
+        public void Run()
+        {
+
+            Console.WriteLine("Reading file...");
+
+            FileLoader.Load(_ontologyGraph, rdfFile.FullName);
+
+            // Start looping through for each owl:Class
+            foreach (OntologyClass owlClass in _ontologyGraph.OwlClasses)
+            {
+
+                // Generate a DTMI for the owl:Class
+                string Id = GenerateDTMI(owlClass);
+
+                if (!String.IsNullOrEmpty(Id))
+                {
+
+                    Console.WriteLine($"{owlClass.ToString()} -> {Id}");
+
+                    // Create Interface
+                    var dtdlInterface = new DtdlInterface
+                    {
+                        Id = Id,
+                        Type = "Interface",
+                        DisplayName = GetInterfaceDisplayName(owlClass),
+                        Comment = GetInterfaceComment(owlClass),
+                        Contents = new List<DtdlContents>(),
+                    };
+
+                    // Use DTDL 'extends' for super classes 
+                    IEnumerable<OntologyClass> foundSuperClasses = owlClass.DirectSuperClasses;
```

Comment:
what is the purpose of this line?

Reply:
Reached out to author (@khilscher) to ask

Reply:
Copying reply from Kevin: "An OWL graph can have parent/child classes. So to understand if an OWL class is a base class or a child class, we can look for a superclass on any given OWL class. If found, we convert these to parent + child Interfaces using DTDL extends."

The full project on which this excerpt is based can be found in this repo: (Azure-Samples/RdfToDtdlConverter)[https://github.com/azure-samples/rdftodtdlconverter/tree/main/]

Reply:
This would make a good code comment.

Reply:
@cippero Can we actually reword the comment slightly to avoid "we" ? (Style guide recommends avoiding it where possible)
Could do this by just removing "we" and "we can" to make it a command

Reply:
Will do